### PR TITLE
Search and replace link to Vim lecture, minor R correction and Rust uninstall clarification per Jon review

### PR DESCRIPTION
**Pull request by hoxiea**
Still a git amateur, unfortunately, so couldn't quite figure out how to make changes to the [former pull request](https://github.com/missing-semester/missing-semester/pull/55). Unless I'm mistaken, this request should contain the changes that @jonhoo and I worked through.

### CODE REVIEW

File: _2020/data-wrangling.md [hunk 1]
```diff
@@ -253,8 +258,8 @@ ssh myserver journalctl
  | awk '{print $2}' | paste -sd,
 ```
 
-Let's start with `paste`: it lets you combine lines (`-s`) by a given
-single-character delimiter (`-d`). But what's this `awk` business?
+Let's start with `paste`: it lets you combine lines (`-s`) separated by a given
```
Comment (jonhoo) [inline _2020/data-wrangling.md:261]: Ah, so, this isn't quite accurate. The way it was is correct — `paste` lets you "combine lines ... by a given delimiter", it does not require those lines to be separated by the given delimiter.

**Comment by jonhoo** [comment]
Thanks! It should just be a matter of comitting your changes and then doing a `git push`, and the PR should update automatically?

**Comment by hoxiea** [comment]
It probably was that simple before I started reseting and reverting, but pushing seems to have just worked a minute ago - great to know, thank you.

**Comment by jonhoo** [comment]
Yup, that seems to have worked, thank you!

**Pull request merged**